spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -433,6 +444,8 @@ class SpdxLicenseIdExpression(
             // it is a generic "+" operator for deprecated licenses.
             if (orLaterVersion && !id.endsWith("-or-later")) append("+")
         }
+
+    override fun getLicenseUrl() = BASE_URL + id
```

Comment:
If you make the function nullable as proposed above, this should be:
`if (isValid()) BASE_URL + id else null`

Reply:
Done it this way but with another var name (`SPDX_BASE_URL`) as proposed above.